### PR TITLE
chore: docs & better apple login on android

### PR DESCRIPTION
**Pull request by WcaleNieWolny**

### CODE REVIEW

File: ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift [hunk 1]
```diff
@@ -44,9 +44,11 @@ public class SocialLoginPlugin: CAPPlugin, CAPBridgedPlugin {
         }
 
         if let appleSettings = call.getObject("apple") {
-            if let appleClientId = appleSettings["clientId"] as? String,
```
Comment (riderx) [inline ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift:47]: Why remove it? ClientID should be defined here not in login

Comment (WcaleNieWolny) [reply ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift:47]: I removed because IOS doesn't use clientID. This variable is 100% useless for IOS.

Comment (riderx) [reply ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift:47]: Ok for Iso but it shouldn’t be define in login for android

Comment (WcaleNieWolny) [reply ios/Sources/SocialLoginPlugin/SocialLoginPlugin.swift:47]: I have no idea what you are talking about
IOS:
image

Android:
image

**Pull request merged**